src/mugs.js

```diff
@@ -667,37 +628,21 @@ define([
                 nodeId = isSelectItem ?
                     this.p.defaultValue || "null" :
                     this.getDefaultItextRoot(),
-                itextType = propertyPath.replace("ItextID", "");
-       
+                itextType = propertyPath.replace("Itext", "");
             return rootId + nodeId + "-" + itextType;
         },
-        setItextId: function (propertyPath, id, unlink) {
-            var itext = this.p[propertyPath];
-
-            if (id !== itext.id) {
-                if (unlink) {
-                    itext = itext.clone();
-                    this.form.vellum.data.javaRosa.Itext.addItem(itext);
-                }
-
-                itext.id = id;
-                // HACK to ensure property really changes
-                this.p.__data[propertyPath] = null;
-                this.p[propertyPath] = itext;
-            }
-        },
         unlinkItext: function () {
             var _this = this;
             _.each([
-                "labelItextID",
-                "constraintMsgItextID",
-                "hintItextID"
+                "labelItext",
+                "constraintMsgItext",
+                "hintItext"
```

Comment:
Looks like we need to add `helpItext` here. And move it over to javaRosa.js too.